staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -38,6 +39,10 @@ const (
 	successLabel = "success"
 	failureLabel = "failure"
 	errorLabel   = "error"
+
+	allowedLabel   = "allowed"
+	forbiddenLabel = "forbidden"
```

Comment:
```suggestion
	deniedLabel = "denied"
```

`deny|denied` is the language used in the Kubernetes [authorization docs](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) and the [API](https://github.com/kubernetes/api/blob/master/authorization/v1/types.go#L176-L193) to refer to this case, although I see that the `forbid|forbidden` language is used in the code.  IMO we should align the metric name with the language in the public facing docs and the API instead of the code.

Reply:
Thanks for the suggestion, changing it to denied seems to be a more reasonable behavior